views/term_table.erb

```diff
@@ -112,12 +112,12 @@
 
                 <span class="person-card-filter">
                     <label for="filter-input" class="fa fa-search"></label>
-                    <input type="text" class="js-filter-input" id="filter-input" placeholder="Search by name, place, etc…">
+                    <input type="text" class="js-filter-input" id="filter-input" placeholder="Search by name, place, etc…" data-ga-track-click="Filter field">
```

Comment:
Should probably be tracking the `input` event here, rather than `click`. Firstly because you can tab to an input without clicking. And secondly because it's the act of actually *changing* the input’s value that we’re interested in tracking.